pyfar/classes/filter.py

```diff
@@ -383,6 +416,33 @@ def init_state(self, cshape, state='zeros'):
                 new_state[idx, ...] = spsignal.lfilter_zi(coeff[0], coeff[1])
         super().init_state(state=new_state)
 
+    def impulse_response(self, n_samples=None):
+        """
+        Compute the finite impulse response of the filter.
+
+        Parameters
```

Comment:
```suggestion
        This method will return a warning if the ``n_samples`` is smaller than the number of coefficients.
        
        Parameters
```
I would like to add the behaviour to the docstring as well.